doc/sphinx/source/build/project_build.rst

```diff
@@ -145,7 +145,7 @@ and generate executable file (.bin/.hex)
 
             * In order to choose STM32 platform in the firmware, select the "ACTIVE_PLATFORM" as "STM32_PLATFORM" in the app_config.h from the respective project.
 
-            * Add compiler flags "-u _printf_float" to the project settings.
+            * Add compiler flags "-u _printf_float" to the project settings. Similarly , add compiler flag "ACTIVE_PLATFORM == 2" for selecting stm platform.
```

Comment:
Also, should we move the bit about Stm32 to Line 146, as they're related. Could say something like "Alternatively, add compiler flag ...." ?